crd/apis/network/v1/annotations.go

```diff
@@ -64,6 +66,20 @@ type InterfaceRef struct {
 	Interface *string `json:"interface,omitempty"`
 }
 
+// NICInfoAnnotation is the value of the nic-info annotation
+type NICInfoAnnotation []NICInfoRef
+
+//NICInfoRef specifies the mapping between a NIC's first v4 IP and its
```

Comment:
can it only be v4? what about v6?

Reply:
We only support ipv4 ATM

Reply:
We can extend a field later for v6, something like birthIP6